packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardTitle.tsx

```diff
@@ -137,14 +134,12 @@ export class DocumentCardTitle extends BaseComponent<IDocumentCardTitleProps, ID
       const { truncatedTitleFirstPiece, truncatedTitleSecondPiece } = this.state;
       this._isTruncated = true;
 
-      if (!truncatedTitleFirstPiece && !truncatedTitleSecondPiece) {
-        this._startTruncation(this.props);
+      if (truncatedTitleFirstPiece || truncatedTitleSecondPiece) {
```

Comment:
Adding this check to stop infinitely calling `setState()` which will cause looping back to the same spot through `componentDidUpdate()`. When the shrinking reached so that both strings are empty stop updating.